products/views.py

```diff
@@ -1,3 +1,41 @@
 from django.shortcuts import render
+from rest_framework import viewsets, mixins, generics
+from .serializers import (ProductSerializer,
+                          ProductAttributeSerializer,
+                          CategorySerializer,
+                          BrandSerializer,
+                          AttributeSerializer)
+from .models import Product, Brand, Category, Attribute, ProductAttribute
+from .permissions import IsAdminUserOrReadOnly
+from .pagination import ProductsResultSetPagination
```

Comment:
Rest framework has default pagination turned on. You can configure it in settings.py
Defining a pagination class is not required if you don't have some specific pagination.
Besides, not only products should be paginated but all list views